scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -36,148 +36,188 @@
-                    <div class="row">
-                      <div class="col">
-                        {{ form.alamut_key.label(class="control-label") }}
-                        {{ form.alamut_key(class='form-control', value=institute.alamut_key if institute.alamut_key) }}
-                      </div>
-                      <div class="col">
-                        {{ form.alamut_institution.label(class="control-label") }}
-                        {{ form.alamut_institution(class='form-control', value=institute.alamut_institution if institute.alamut_institution) }}
-                      </div>
-                    </div>
-                  </fieldset>
+
+                <div class="col-2">
+                  <input type="checkbox" name="check_show_all_vars" {% if institute.check_show_all_vars %} checked {%endif%}>
+                  {{ form.check_show_all_vars.label(class="control-label", data_bs_toggle="tooltip", data_bs_placement="top", title="Check this box if you want all variants (and not just those occurring in the affected individuals) to be shown by default on variants page") }}
                 </div>
-                <!-- End of Alamut settings -->
-              {% endif %}
-            </div>
 
-            <div class="row mt-3">
-              <div class="col">
-                <input type="checkbox" name="check_show_all_vars" {% if institute.check_show_all_vars %} checked {%endif%}>
-                {{ form.check_show_all_vars.label(class="control-label", data_bs_toggle="tooltip", data_bs_placement="top", title="Check this box if you want all variants (and not just those occurring in the affected individuals) to be shown by default on variants page") }}
               </div>
+
+            </fieldset>
             </div>
+            <!-- End of Variants and panels searching -->
+
+
+            {% if "admin" in current_user.roles %}
+              <br>
+              <!-- LoqusDB settings -->
+              <div class="row mt-3 d-flex align-items-center">
+                <fieldset>
+                  <legend>LoqusDB</legend>
+                  <div class="row">
+                    <div class="col-2">
+                      {{ form.loqusdb_id.label(class="control-label", data_bs_toggle="tooltip", data_bs_placement="top", title="LoqudDB id" ) }}
+                    </div>
+                    <div class="col-3">
+                      <select multiple="multiple" class="selectpicker" name="loqusdb_id" data-style="btn-secondary">
+                        {% for instance in loqus_instances %}
+                          <option value="{{instance}}" {% if instance in institute.loqusdb_id %} selected {% endif %}>{{instance}}</option>
+                        {% endfor %}
+                      </select>
+                    </div>
+                  </div>
+              </div>
+              <!-- End of loqusdb settings -->
+
+              <br>
```

Comment:
As before! 😊

Reply:
Ah, or are the `row` plus explicit `d-flex` stuff causing some issue?

Reply:
No issue, I just wanted to give it an extra space

Reply:
Ok! It's just layout so whatever, but looks cleaner with just a larger margin, like mb-4 or mb-5, doesn't it? BR is fine if you prefer it of course, but then perhaps drop the margins on the rows altogether?

Reply:
No it's a good suggestion. I've replaced the newline with mt-5 and looks the same, thanks!